streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -385,30 +387,30 @@ public void shouldConstructMetricsWithBuiltInMetricsVersionLatest() {
     private void testMetrics(final String builtInMetricsVersion) {
         task = createStatelessTask(createConfig(false, "100"), builtInMetricsVersion);
 
-        assertNotNull(getMetric(
-            "commit",
-            "%s-latency-avg",
-            task.id().toString(),
-            builtInMetricsVersion
-        ));
-        assertNotNull(getMetric(
-            "commit",
-            "%s-latency-max",
-            task.id().toString(),
-            builtInMetricsVersion
-        ));
-        assertNotNull(getMetric(
-            "commit",
-            "%s-rate",
-            task.id().toString(),
-            builtInMetricsVersion
-        ));
-        assertNotNull(getMetric(
-            "commit",
-            "%s-total",
-            task.id().toString(),
-            builtInMetricsVersion
-        ));
+//        assertNotNull(getMetric(
```

Comment:
After we addressed the question how we want to do metrics, we can update this tests